src/store/getters/state-getters.ts

```diff
@@ -49,6 +48,17 @@ export const isAlterationFiling = (state: StateIF): boolean => {
   return (state.stateModel.tombstone.filingType === FilingTypes.ALTERATION)
 }
 
+/** Whether the current filing is a change filing. */
+export const isChangeFiling = (state: StateIF): boolean => {
+  return (state.stateModel.tombstone.filingType === FilingTypes.CHANGE_FIRM)
+}
+
+/** Whether the current filing is an Edit filing. */
+export const isEditFiling = (state: StateIF): boolean => {
+  return (state.stateModel.tombstone.filingType === FilingTypes.CHANGE_FIRM) ||
+    (state.stateModel.tombstone.filingType === FilingTypes.ALTERATION)
+}
```

Comment:
Naming things is hard.

I originally wrote that I didn't truly understand the difference between a "change" filing and an "edit" filing. And after looking closer, I see that I was even more confused than I thought. I see now that an edit filing is a superset of a change filing. My spidey senses are tingling!

I recommend that you replace isEditFiling with `isChangeFiling || isAlterationFiling`. This is more verbose but also more clear.

Reply:
Can do but it also brings into question the directory structure..

Reply:
I think we'd be better with more verbosity (clarity) instead of optimization here. Honestly it's a gray area and more of a "feel" than anything else.

Or rename this getter `isChangeAlterationFiling` ??? (IMO this is ugly but clear. Prefer you don't do this 😁 )

Reply:
Oh for sure, will refactor this.
We don't need to centralize this computed value. `isChangeFiling || isAlterationFiling` is fine!

Reply:
Updated!